container/webui/conf/client.conf

```
@@ -1,3 +1,7 @@
+[localhost]
+key = 1234567890ABCDEF
+secret = 1234567890ABCDEF
+
 [scheduler]
```

Maybe you can throw together a quick hack to instead of needing to support to duplicate all these sections to define something like this:

```suggestion
[localhost,scheduler]
```

---

Allowing this syntax would be a good idea. It'll likely be easy to achieve so I can give it a try later (or tomorrow). However, I'd like to have it coded in a clean way and see no reason to rush it.